neurots/generate/algorithms/basicgrower.py

```diff
@@ -36,10 +38,24 @@ class TrunkAlgo(AbstractAlgo):
         context (Any): An object containing contextual information.
     """
 
-    def __init__(self, input_data, params, start_point, context=None, **_):
+    def __init__(
+        self,
+        input_data,
+        params,
+        start_point,
+        context=None,
+        pia_direction=None,
```

Comment:
Ok, correction to the previous comment. I see that this passes at the neuron level through the parameters. In this case, I would ask why you need to pass it outside here. If I understand correctly it is at a higher level so it is not accessible from the input parameters here (at the tree level). Is this correct? If that's the case, then it makes sense to add here.